CONTRIBUTING.md

```diff
@@ -199,3 +200,34 @@ You may want to use [Github search](https://help.github.com/articles/searching-i
 | Next Release    | [search](https://github.com/search?q=is%3Aopen+is%3Aissue+repo%3Aliferay%2Fclay+label%3A%22status%3A+next-release%22&type=Issues)  | Status     | Will be available in the next release                                                                                                                                                                                                                                               |
 | Duplicate       | [search](https://github.com/search?q=is%3Aopen+is%3Aissue+repo%3Aliferay%2Fclay+label%3A%22resolution%3A+duplicate%22&type=Issues) | Resolution | Duplicates an existing issue                                                                                                                                                                                                                                                        |
 | Wontfix         | [search](https://github.com/search?q=is%3Aopen+is%3Aissue+repo%3Aliferay%2Fclay+label%3A%22resolution%3A+wontfix%22&type=Issues)   | Resolution | Will no longer be worked on                                                                                                                                                                                                                                                         |
+
+## Release process
+
+To publish a new version for all packages that have updated, follow these steps:
+
+1.  `git checkout master`
+2.  Run `lerna version --conventional-commits` and verify each package's version change.
```

Comment:
@bryceosterhaus just out of curiosity, it will set the next version based on type, `feat`, `fix`... maybe we should try to use it more carefully and make sure a `feat` is attached to adding a new API. Just in case lerna gets confused and raises a very tall version.

Reply:
Yeah you're right, we will need to pay closer attention to our commit messages. The other option is to not use `--conventional-commits` but that might be a bit harder to manage all at once. Additionally, we lerna may say "bump to 3.1.0" since there is a "feat" message, but we can override and just say use a patch. So maybe for the first handful of releases we just pay extra close attention.

Reply:
Yeah, sounds good.